app/views/posts/_expanded.html.erb

```diff
@@ -371,9 +371,9 @@
                 </div>
                 <div class="grid--cell is-flexible">
                   <label class="form-element has-margin-0" for="flag-reason-other_<%= post.id  %>">
-                    other reason
+                    moderator attention <br/>
```

Comment:
I like the simplicity of just "moderator attention" rather than padding it out with extra words like "it needs moderator attention".

Would this be a good opportunity to make the other options consistent with this? Currently the first two start with "it's", the third starts with "is", then the rest don't have any extra words. Consistency is my main aim, but my personal preference would be the shortest form, with no "it's" or "is".

I'd be happy to approve this pull request with either consistency applied here, or a separate issue raised for applying the consistency later.

Reply:
I agree we should make it consistent.  My only concern is the issue I hit in https://collab.codidact.org/posts/291874 -- when we push those changes to the network, are we going to get *two* versions of each of those flags?  The only reason it didn't happen here with other -> moderator attention is that that case is baked into the code, not in the YAML file that gets seeded.

I'd like to make the flag text consistent, if we can do it without making a mess.  I don't know how to do that yet. :-(

Reply:
I don't see any reason for this to hold up this pull request. It's not making anything worse. If an issue can be raised then we can work it out when we get to it, and just focus on what's essential for this pull request.

Reply:
Let's file a separate issue for the wording inconsistencies.

Reply:
Do you want to file an issue for that?  Can we resolve this conversation and approve?

Reply:
Added #1359.